vnext/Microsoft.ReactNative/Modules/AlertModule.h

```diff
@@ -26,6 +26,9 @@ struct ShowAlertArgs {
 
   REACT_FIELD(buttonNeutral)
   std::string buttonNeutral;
+
+  REACT_FIELD(defaultButton)
+  int defaultButton;
```

Comment:
this is a good change - I would like the API to be addressed in Core though. Can you drive that? 
https://reactnative.dev/docs/alert#buttons

Reply:
Oh, actually there's already a style value we can piggy-back on.